specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2019-04-01-preview/managedservices.json

```diff
@@ -0,0 +1,902 @@
+            "schema": {
+              "$ref": "#/definitions/RegistrationDefinition"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok - Returns information about the updated registration definition.",
+            "schema": {
+              "$ref": "#/definitions/RegistrationDefinition"
+            }
+          },
+          "201": {
+            "description": "Created - Returns information about the created registration definition.",
+            "schema": {
+              "$ref": "#/definitions/RegistrationDefinition"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Put Registration Definition": {
+            "$ref": "./examples/PutRegistrationDefinition.json"
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.ManagedServices/registrationAssignments/{registrationAssignmentId}": {
+      "get": {
+        "tags": [
+          "RegistrationAssignments"
+        ],
+        "operationId": "RegistrationAssignments_Get",
+        "description": "Gets the details of specified registration assignment.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistrationAssignmentIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ExpandRegistrationDefinition"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns the registration assignment.",
+            "schema": {
+              "$ref": "#/definitions/RegistrationAssignment"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Registration Assignment": {
+            "$ref": "./examples/GetRegistrationAssignment.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "RegistrationAssignments"
+        ],
+        "operationId": "RegistrationAssignments_Delete",
+        "x-ms-long-running-operation": true,
+        "description": "Deletes the specified registration assignment.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistrationAssignmentIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
```

Comment:
200 OK response also needs to be modeled. When the long running operation completes, GET on the location header will return 200 OK. the client needs to know what is the final response of this operation. Hence it should be modeled

Reply:
Added 200 also.